templates/agenda-for-all-hands-call.md

```diff
@@ -7,7 +7,7 @@
 
 
 ## Agenda
-1. Roll call by alphabetical order (moderator)
+1. Roll call - see [timezone-based roll call](../admin-guides/timezone-rollcall.md) guide - _ensure notetaker is present_
```

Comment:
You pick the notetaker & moderator at the end of the prior week's call. This is a reminder to make sure that person is on the call before starting the roll call.